R/transform.R

```diff
@@ -0,0 +1,27 @@
+#' Transform touchstone-relevant extracted data into tables ready for
+#' database import. This should be called from the transform code of montagu-import,
+#' calling `stoner::transform(extracted_data)` with the same argument provided
+#' to the montagu-import transform function.
+#'
+#' For specific information on what steps the transform function does.
+#' see the vignette.
+#' @title Transform extracted_data into transformed_data
+#' @export
+#' @param extracted_data The extracted data that was generated by the previous extract call
+#' @return A list of data frames, one for each table, with names matching the
+#'   database table names, and containing rows to be added to that table.
+
+transform <- function(extracted_data) {
+  t <- c(transform_touchstone(extracted_data))
```

Comment:
What is `c` being used for here?

Reply:
`transform_touchstone(extracted_data)` returns a list of zero, one, or two data frames. Future PRs will also return lists of zero, one or more data frames, and I want the result to be a flat list of all the elements.